tests/OSPSuite.Core.Tests/Domain/FormulaTaskSpecs.cs

```diff
@@ -549,6 +553,9 @@ protected override void Context()
          _objectPath = new FormulaUsablePath(ORGANISM, _smallIntestine.Name, _mucosa.Name, _duodenumMucosa.Name, LUMEN_SEGMENT, _volumeDuodenumLumen.Name) {Alias = "V"};
          _parameterReferencingAbsoluteLumenSegment.Formula.AddObjectPath(_objectPath);
          _parameterReferencingAbsoluteLumenSegment.Formula.AddObjectPath(new FormulaUsablePath(ORGANISM, _height.Name) {Alias = "H"});
+
+         //to mimic what happens in the model construction, we should expand keywords
+         _keywordReplacerTask.ReplaceIn(_model.Root);
```

Comment:
I had ommited this believing this was not required, but it is if I want to replicate exactly what
s going on when model is being built